main.py

```
@@ -1,11 +1,273 @@
+from rss_controller import RSSController
+from rss_repository import RSSRepository
+
+class ClickableComicWidget(QWidget):
+    clicked = Signal()
+
+    def __init__(self, title: str, pixmap: QPixmap, img_width = 20, img_height=20, parent=None):
+        super().__init__(parent)
+
+        layout = QVBoxLayout(self)
+        layout.setAlignment(Qt.AlignCenter)
+        layout.setContentsMargins
```

Ruff (0.11.9)

Found useless expression. Either assign it to a variable or remove it.

(B018)